test/Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -767,8 +767,8 @@ spec_with name create_new_datetime parse_datetime nanoseconds_loss_in_precision=
         pending_date_diff_test = if loose_zone_equal "" "" then "Loose Zone conversions are on, skipping the test"
 
         Test.specify "should allow computing date_diff" pending=pending_date_diff_test <|
-            t1 = create_new_datetime 2021 11 3 10 15 0
-            t2 = create_new_datetime 2021 12 5 12 30 20
+            t1 = create_new_datetime 2021 11 3 10 15 0 zone=Time_Zone.utc
+            t2 = create_new_datetime 2021 12 5 12 30 20 zone=Time_Zone.utc
```

Comment:
I'm wondering if instead of this, we should just make the default `zone` be `utc` in the test builders.

i.e.
```diff
Index: test/Tests/src/Data/Time/Date_Time_Spec.enso
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test/Tests/src/Data/Time/Date_Time_Spec.enso b/test/Tests/src/Data/Time/Date_Time_Spec.enso
--- a/test/Tests/src/Data/Time/Date_Time_Spec.enso	(revision eb1550014b3c7120dc57ab111921eb7d18c2d5fe)
+++ b/test/Tests/src/Data/Time/Date_Time_Spec.enso	(date 1692868316355)
@@ -847,7 +847,7 @@
 
     Date_Part_Spec.spec name create_new_datetime
 
-js_datetime year month=1 day=1 hour=0 minute=0 second=0 nanosecond=0 zone=Time_Zone.system =
+js_datetime year month=1 day=1 hour=0 minute=0 second=0 nanosecond=0 zone=Time_Zone.utc =
     Panic.catch Any (js_datetime_with_zone year month day hour minute second nanosecond zone) (err -> Error.throw (Time_Error.Error err.payload))
 
 # This ensures that date returned by javascript has the right timezone specified by the zone parameter.
@@ -914,10 +914,10 @@
     else:
         return datetime.datetime(year, month, day, hour, minute, second, microseconds, Zone(zone, zone_name))
 
-enso_datetime year month=1 day=1 hour=0 minute=0 second=0 nanosecond=0 zone=Time_Zone.system =
+enso_datetime year month=1 day=1 hour=0 minute=0 second=0 nanosecond=0 zone=Time_Zone.utc =
     Date_Time.new year month day hour minute second nanosecond=nanosecond zone=zone
 
-java_datetime year month=1 day=1 hour=0 minute=0 second=0 nanosecond=0 zone=Time_Zone.system =
+java_datetime year month=1 day=1 hour=0 minute=0 second=0 nanosecond=0 zone=Time_Zone.utc =
     Panic.catch Any (ZonedDateTime.of year month day hour minute second nanosecond zone) (err -> Error.throw (Time_Error.Error <| err.payload.getMessage))
 
 maybe_parse_java_zoned text pattern=Nothing =
```

What do you think? This way new tests will be more likely to be written correctly.

Reply:
I'll try it, but the test is currently failing for me on develop, for other reasons: https://github.com/enso-org/enso/issues/7655

Reply:
Using this exact diff, I get 18 failures.